go-controller/pkg/util/kube.go

```diff
@@ -249,13 +250,10 @@ func UseEndpointSlices(kubeClient kubernetes.Interface) bool {
 		klog.V(2).Infof("Kubernetes Endpoint Slices enabled on the cluster: %s", discovery.SchemeGroupVersion.String())
 		endpointSlicesEnabled = true
 	}
-	// We only use Slices if > 1.19 since we only need them for Dual Stack
-	sv, _ := kubeClient.Discovery().ServerVersion()
```

Comment:
This method fails in Openshift CI, I think that because the server version has `dirty` .
Anyway, we just care that the service returned has the `ClusterIPs` populated, so we check the well known service `kubernetes` in the default namespace